eth-brownie/brownie
# soft and hard requirements

### What I did

Related issue: #991

### How I did it

Added an environment variable to allow the use of soft instead of hard pins.

### How to verify it

`pip install .` and see that everything has hard pins

`BROWNIE_LIB=1 pip install .` and see that everything has soft pins.

### Checklist
